src/applications/disability-benefits/2346/reducers/form2346Reducer.js

```diff
@@ -0,0 +1,41 @@
+import {
+  FETCH_DATA_SUCCESS,
+  FETCH_DATA_FAILURE,
+  UPDATE_DATA_FAILURE,
+  UPDATE_DATA_SUCCESS,
+} from '../constants';
+
+const initialState = {
+  data: [],
+};
+
+export default (state = initialState, action) => {
+  switch (action.type) {
+    case FETCH_DATA_SUCCESS:
+      return {
+        ...state,
+        data: action.payload,
+      };
+
+    case FETCH_DATA_FAILURE:
+      return {
+        ...state,
+        error: action.payload,
```

Comment:
I don't think you've passed a `payload` property in this action: https://github.com/department-of-veterans-affairs/vets-website/pull/11532/files/e518789d389e23af798857b5f6548ee262c3a6ad#diff-24a84f93100c8b19c08ee9cec3ddf07fR14

Might want to name it `error` instead, too 🙂